### ISSUE TITLE
`pod try` Support

### ISSUE THREAD

**Issue by JakeSc** [HarshilShah/DeckTransition#6]
```
$ pod try DeckTransition
Updating spec repositories

Trying DeckTransition
Performing CocoaPods Installation
[!] Unable to find a target named `DeckTransition_Tests`, did find `DeckTransition_Example`.

[!] Automatically assigning platform 
```

**Comment by HarshilShah** [comment]
Hey Jake, thanks for pointing out this issue. Will be fixing it ASAP.

**Comment by HarshilShah** [comment]
This has been fixed in #17, and DeckTransition will be available for pod try from the next release version
